expan/core/statistics.py

```diff
@@ -318,129 +228,50 @@ def bootstrap(x, y, func=_delta_mean, nruns=10000, percentiles=[2.5, 97.5],
 
 
 def pooled_std(std1, n1, std2, n2):
-    """
-    Returns the pooled estimate of standard deviation. Assumes that population
-    variances are equal (std(v1)**2==std(v2)**2) - this assumption is checked
-    for reasonableness and an exception is raised if this is strongly violated.
-
-    Args:
-        std1 (float): standard deviation of first sample
-        n1 (integer): size of first sample
-        std2 (float): standard deviation of second sample
-        n2 (integer): size of second sample
-
-    Returns:
-        float: Pooled standard deviation
+    """ Returns the pooled estimate of standard deviation. 
 
     For further information visit:
         http://sphweb.bumc.bu.edu/otlt/MPH-Modules/BS/BS704_Confidence_Intervals/BS704_Confidence_Intervals5.html
 
-    Todo:
-        Also implement a version for unequal variances.
+    :param std1: standard deviation of first sample
+    :type  std1: float
+    :param n1: size of first sample
+    :type  n1: int
+    :param std12: standard deviation of second sample
+    :type  std2: float
+    :param n2: size of second sample
+    :type  n2: int
+
+    :return: pooled standard deviation
+    :type: float
     """
     if not (0.5 < (std1 ** 2) / (std2 ** 2) < 2.):
-        warnings.warn('Sample variances differ too much to assume that '
-                      'population variances are equal.')
+        warnings.warn('Sample variances differ too much to assume that population variances are equal.')
```

Comment:
Do we need this warning in scalyr? Maybe it make sense to add it to logger?

Reply:
good point